eZ/Bundle/EzPublishRestBundle/Resources/config/routing.yml

```diff
@@ -773,7 +773,7 @@ ezpublish_rest_loadRoleDraft:
 ezpublish_rest_loadRoleDraftByRoleId:
     path: /user/roles/{roleId}/draftByRoleId
```

Comment:
I'd actually have expected this URI to be `/user/roles/{roleId}/draft`. The "byRoleId" part isn't needed, as the URI already contains the fact that we are using an ID.